src/ktopt_planning_context.cpp

```diff
@@ -137,19 +121,38 @@ void KTOptPlanningContext::solve(planning_interface::MotionPlanResponse& res)
     return;
   }
 
+  RCLCPP_INFO(getLogger(), "Setting initial guess ...");
+  // set the initial guess
+  trajopt.SetInitialGuess(trajopt.ReconstructTrajectory(result));
+  // const auto& collision_constraint = MinimumDistanceLowerBoundConstraint(&plant_, 0.01, &plant_context_);
```

Comment:
I think you can remove this now since the for-loop handles the collision constraints.